pkg/js/helpers.js

```diff
@@ -156,6 +170,18 @@ function ALIAS(name, target) {
     }
 }
 
+// CAA(name,tag,value, recordModifiers...)
+function CAA(name, tag, value){
+    checkArgs([_.isString, _.isString, _.isString], arguments, "CAA expects (name, tag, value)")
```

Comment:
I like the integer idea, even though I kinda hate the MX special case. 

I like the user-friendliness of `CAA("@", "tag", "target", {meta: "whatever"},  CAA_CRITICAL | CAA_FUTURE_USE)`. Or alternatively, have flags be an arg to a function:

`CAA("@", "tag", "target", {meta: "whatever"},  CAA_FLAGS(CAA_CRITICAL | CAA_FUTURE_USE))`